nixos/modules/services/web-servers/gunicorn/instance-options.nix

```
@@ -0,0 +1,112 @@
+{
+  config,
+  lib,
+  ...
+}:
+
+{
+  options = {
+    app.package = lib.mkOption {
+      type = lib.types.package;
+      description = "Package of the WSGI Python application.";
+    };
+
+    app.python = lib.mkOption {
+      type = lib.types.package;
+      default = config.app.package.python;
+      defaultText = lib.literalExpression "services.gunicorn.instances.<name>.app.package.python";
+      description = ''
+        Python interpreter package to use.
+        Defaults to the one exposed by the WSGI application package.
+      '';
+    };
+
+    app.module = lib.mkOption {
+      type = lib.types.str;
+      default = config.app.package.wsgiModule;
+      defaultText = lib.literalExpression "services.gunicorn.instances.<name>.app.package.wsgiModule";
```

Is this assuming that the package sets `passthru.wsgiModule`? If so, it might be a good addition to the `description`.

```suggestion
      defaultText = lib.literalExpression "config.services.gunicorn.instances.<name>.app.package.wsgiModule";
```